test/integration/basic_delete_dirty_nse_test.go

```diff
@@ -0,0 +1,44 @@
+// +build basic
+
+package nsmd_integration_tests
+
+import (
+	"github.com/networkservicemesh/networkservicemesh/test/integration/nsmd_test_utils"
+	"testing"
+	"time"
+
+	"github.com/networkservicemesh/networkservicemesh/test/kube_testing"
+	"github.com/networkservicemesh/networkservicemesh/test/kube_testing/pods"
+
+	. "github.com/onsi/gomega"
+	"github.com/sirupsen/logrus"
+)
+
+func TestDeleteDirtyNSE(t *testing.T) {
+	RegisterTestingT(t)
+
+	if testing.Short() {
+		t.Skip("Skip, please run without -short")
+		return
+	}
+
+	logrus.Print("Running NSE Delete test")
+
+	k8s, err := kube_testing.NewK8s()
+	defer k8s.Cleanup()
+
+	Expect(err).To(BeNil())
+
+	k8s.PrepareDefault()
+	nodes := nsmd_test_utils.SetupNodesConfig(k8s, 1, defaultTimeout, []*pods.NSMgrPodConfig{})
+
+	nsePod := nsmd_test_utils.DeployDirtyNSE(k8s, nodes[0].Node, "dirty", defaultTimeout)
+	k8s.DeletePods(nsePod)
+
+	time.Sleep(5 * time.Second)
```

Comment:
And here?

Reply:
Fixed : [place](https://github.com/Bolodya1997/networkservicemesh/blob/nsm-monitor-nse/test/integration/basic_delete_dirty_nse_test.go#L73)